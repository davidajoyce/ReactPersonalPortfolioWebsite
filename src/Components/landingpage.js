import React, { Component } from 'react';
import { Grid, Cell } from 'react-mdl';

class LandingPage extends Component{
  render(){
      return(
        <div className="landing-body">
          <Grid className="landing-grid">
            <Cell col={12}>
              <img
                src="https://drive.google.com/uc?export=view&id=1g0LIW9mrYH4Nq17CemHdx3IEXU9QQR1C"
                alt="avatar"
                className="avatar-img"
                />
                <div className="banner-text">
                   <h1>Software Engineer</h1>

                <hr/>
                <p>Java | Spring | Hibernate |C# | Python | React | SQL | AWS | </p>

              <div className="social-links">

                {/*LinkedIn*/}
                <a href="https://www.linkedin.com/in/david-joyce-01397296/" rel="noopener noreferrer" target="_blank">
                  <i className="fa fa-linkedin-square" aria-hidden="true" />
                </a>

                {/*Github*/}
                <a href="https://github.com/davidajoyce" rel="noopener noreferrer" target="_blank">
                  <i className="fa fa-github-square" aria-hidden="true" />
                </a>

              </div>
                </div>
            </Cell>
          </Grid>
        </div>
      )
    }
}

export default LandingPage;
