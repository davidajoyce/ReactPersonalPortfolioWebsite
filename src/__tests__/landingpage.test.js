import React from 'react'
import landingpage from '../Components/landingpage'
import { shallow } from 'enzyme';

describe('landingpage', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<landingpage />);
    expect(wrapper).toMatchSnapshot();
    // On the first run of this test, Jest will generate a snapshot file automatically.
  });
});
