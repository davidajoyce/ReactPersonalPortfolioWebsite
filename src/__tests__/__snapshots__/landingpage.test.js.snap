// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landingpage renders correctly 1`] = `ShallowWrapper {}`;
